addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/environment.html

```diff
@@ -65,9 +65,9 @@ <H1>Automation Framework - Environment</H1>
           password:                    # String, the password to use when authenticating, vars supported
   vars:                                # List of 0 or more custom variables to be used throughout the config file
     myVarOne: CustomConfigVarOne       # Can be used as ${myVarOne} anywhere throughout the config
-    myVarTwo: ${myVarOne}.VarTwo       # Can refer other vars    
+    myVarTwo: ${myVarOne}.VarTwo       # Can refer other vars
   parameters:
-    failOnError: true                  # If set exit on an error         
+    failOnError: true                  # If set exit on an error
```

Comment:
Yes these white spaces changes are unrelated but it looked horrible, could we just go with it :grinning: